orbs/pii/orb.yml

```diff
@@ -0,0 +1,339 @@
+version: 2.1
+
+description: |
+  Run pii tests from privacy and e2e_jt_test repositories
+
+orbs:
+  go-service: "jobteaser/go-service@0.36.0"
+  service: "jobteaser/service@0.10.0"
+  browser-tools: circleci/browser-tools@1.4.0
+  slack: "circleci/slack@4.10.1"
+
+executors:
+  xlarge:
+    parameters:
+      max_instances:
+        type: string
+        default: "30"
+    resource_class: "xlarge"
+    docker:
+      - image: "cimg/go:1.19.3-browsers"
+        environment:
+          DEBUG: "trace"
+          E2E_MAX_INSTANCES: << parameters.max_instances >>
+          FORCE_COLOR: "0"
+  large:
+    parameters:
+      max_instances:
+        type: string
+        default: "30"
+    resource_class: "large"
+    docker:
+      - image: "cimg/go:1.19.3-browsers"
+        environment:
+          DEBUG: "trace"
+          E2E_MAX_INSTANCES: << parameters.max_instances >>
+          FORCE_COLOR: "0"
+  medium:
+    parameters:
+      max_instances:
+        type: string
+        default: "30"
+    docker:
+      - image: "cimg/go:1.19.3-browsers"
+        environment:
+          DEBUG: "trace"
+          E2E_MAX_INSTANCES: << parameters.max_instances >>
+          FORCE_COLOR: "0"
+  small:
+    parameters:
+      max_instances:
+        type: string
+        default: "30"
+    resource_class: "small"
+    docker:
+      - image: "cimg/go:1.19.3-browsers"
+        environment:
+          DEBUG: "trace"
+          E2E_MAX_INSTANCES: << parameters.max_instances >>
+          FORCE_COLOR: "0"
+
+commands:
+  clone_e2e_jt_tests:
+    description: "Clone e2e_jt_tests repository"
+    steps:
+      - run:
+          name: "Clone e2e_jt_tests repository"
+          command: |
+            git clone -b master git@github.com:jobteaser/e2e_jt_tests.git
+          working_directory: "~/project"
+
+  clone_privacy:
+    description: "Clone privacy repository"
+    steps:
+      - run:
+          name: "Clone privacy repository"
+          command: |
+            git clone -b master git@github.com:jobteaser/privacy.git
```

Comment:
**Reminder: ⚠️  All code in the present repo is public.**

Please remove this and make the repository URL a parameter.

Reply:
Info from Yohan:
`Hello Hervé, après quelques discussions, on va de notre côté essayer de passer le repo circleci en privé, mais ça va prendre un peu de temps. En attendant, on a donné le feu vert pour merger la PR en l'état
`